.github/workflows/tests.yml

```diff
@@ -110,10 +119,6 @@ jobs:
     steps:
       - name: "Conan cache"
         uses: faasm/conan-cache-action@v1
-        with:
-           directory: /usr/local/code/faasm
-      - name: "Build dev dependencies to be shared by all runs"
-        run: ./bin/inv_wrapper.sh dev.cmake --build Debug
```

Comment:
Without running CMake this job becomes useless, the cache action only sets up the cache, while cmake populates it when it runs `conan install` during configuration.

Reply:
Thats a good point, thanks for bringing this up. The reason I removed it in the first place was because the `CMakeCache` would point to a different directory (so we needed the `--clean` and I thought that if we were to build with `--clean` then we might as well do it just once) but you are right.